### DISCUSSION TITLE
What is the point of watch from useForm() when useWatch exists?

### DISCUSSION THREAD

**Discussion by mvandoff-avi** [react-hook-form/react-hook-form#9111] [General]
It seems that `watch`  is the same as `useWatch` but with better performance? Is there any reason to ever use the former?

**Comment by bluebill1049** [comment]
refer to the doc: https://react-hook-form.com/api/useform/watch i made a video on the API as well.

> This API will trigger re-render at the root of your app or form, consider using a callback or the [useWatch](https://react-hook-form.com/api/usewatch) api if you are experiencing performance issues.
